pages/common/ngs.md

```diff
@@ -0,0 +1,16 @@
+# ngs
+
+> Fully featured scripting language created specifically for Ops. REPL is being developed.
```

Comment:
This information might be outdated in the future. `Fully featured` is subjective and doesn't help more clearly define what the command actually is or does.
```suggestion
> Scripting language created specifically for Ops.
```